### PR TITLE
Update dotnet-grpc.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/grpc/dotnet-grpc?view=aspnetcore-3.0&branch=pr-en-us-14935)

Merges into #14565

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@JunTaoLuo  let me know what you want changed, then I'll merge this into your PR.

* Shortened sentences
* Removed future tense.

**Comment by Rick-Anderson** [comment]
@JunTaoLuo let me know what to change before merging into #14565

File: aspnetcore/grpc/dotnet-grpc.md [hunk 1]
```diff
@@ -23,17 +23,28 @@ dotnet tool install -g dotnet-grpc
...
 ```xml
 <Protobuf Include="..\Proto\count.proto" GrpcServices="Server" Link="Protos\count.proto" />
 ```
 
-The Protobuf references are used to generate the C# client and/or server assets. The tool can be used to create a Protobuf reference from local file(s) on disk or a remote file specified by an URL. The tool will also ensure the correct gRPC package dependencies are added to the project. For example, the `Grpc.AspNetCore` package, which contains gRPC server and client libraries as well as tooling support, will be added to a Web app. Alternatively, the `Grpc.Net.Client`, `Grpc.Tools` and `Google.Protobuf` packages, which contains only the gRPC client libraries and tooling support, will be added to a Console app.
+The Protobuf references are used to generate the C# client and/or server assets. The `dotnet-grpc`tool can:
 
-### Add file 
+* Create a Protobuf reference from local files on disk.
+* Create a Protobuf reference from  a remote file specified by a URL.
```
Comment (JunTaoLuo) [inline aspnetcore/grpc/dotnet-grpc.md]:
```suggestion
* Create a Protobuf reference from a remote file specified by a URL.
```

File: aspnetcore/grpc/dotnet-grpc.md [hunk 2]
```diff
@@ -23,17 +23,28 @@ dotnet tool install -g dotnet-grpc
...
+* Ensure the correct gRPC package dependencies are added to the project.
 
-The `add-file` command is used to add local file(s) on disk as Protobuf reference(s). The file path(s) provided can be relative to the current directory or absolute paths and may contain wild cards for pattern based file globbing. If any file(s) are outside the project directory, a `Link` element will be added to display the file under the folder `Protos` in Visual Studio.
+For example, the `Grpc.AspNetCore` package is added to a web app. `Grpc.AspNetCore` contains gRPC server and client libraries and tooling support. Alternatively, the `Grpc.Net.Client`, `Grpc.Tools` and `Google.Protobuf` packages, which contain only the gRPC client libraries and tooling support, are added to a Console app.
+
+### Add file
+
+The `add-file` command is used to add local files on disk as a Protobuf references. The file paths provided:
```
Comment (JunTaoLuo) [inline aspnetcore/grpc/dotnet-grpc.md]:
```suggestion
The `add-file` command is used to add local files on disk as Protobuf references. The file paths provided:
```

File: aspnetcore/grpc/dotnet-grpc.md [hunk 3]
```diff
@@ -45,20 +56,20 @@ dotnet grpc add-file [options] <files>...
 
 | Argument | Description |
 |-|-|
-| files | The protobuf file reference(s). These can be a path to glob for local protobuf file(s). |
+| files | The protobuf file references. These can be a path to glob for local protobuf files. |
```
Comment (JunTaoLuo) [inline aspnetcore/grpc/dotnet-grpc.md]: I lifted these from the CLI help text directly. Is the recommendation to update the help text? @shirhatti what do you think?

File: aspnetcore/grpc/dotnet-grpc.md [hunk 4]
```diff
@@ -45,20 +56,20 @@ dotnet grpc add-file [options] <files>...
...
 
 #### Options
 
 | Short option | Long option | Description |
 |-|-|-|
-| -p | --project | The path to the project file to operate on. If a file is not specified, the command will search the current directory for one.
-| -s | --services | The type of gRPC services that should be generated. If Default is specified, Both will be used for Web projects and Client will be used for non-Web projects. Accepted values are Both, Client, Default, None, Server.
+| -p | --project | The path to the project file to operate on. If a file is not specified, the command searchs the current directory for one.
```
Comment (JunTaoLuo) [inline aspnetcore/grpc/dotnet-grpc.md]:
```suggestion
| -p | --project | The path to the project file to operate on. If a file is not specified, the command searches the current directory for one.
```

File: aspnetcore/grpc/dotnet-grpc.md [hunk 5]
```diff
@@ -72,20 +83,22 @@ dotnet-grpc add-url [options] <url>
...
-| | --access | The access modifier to use for the generated C# classes. Default value is Public. Accepted values are Internal and Public.
+| | --access | The access modifier to use for the generated C# classes. Default value is `Public`. Accepted values are `Internal` and `Public`.
 
 ## Remove
 
-The `remove` command is used to remove Protobuf references from the *.csproj* file. The command accepts path(s) arguments and source URL as arguments. Note that the tool only removes the Protobuf reference but the actual *.proto* file is not deleted, even if it were originally downloaded from a remote URL.
+The `remove` command is used to remove Protobuf references from the *.csproj* file. The command accepts path arguments and source URLs as arguments. The tool:
+
+* Only removes the Protobuf reference.
+* The actual *.proto* file is not deleted, even if it was originally downloaded from a remote URL.
```
Comment (JunTaoLuo) [inline aspnetcore/grpc/dotnet-grpc.md]:
```suggestion
* Does not delete the *.proto* file, even if it was originally downloaded from a remote URL.
```

**Pull request merged**